adagrams/game.py

```diff
@@ -1,11 +1,154 @@
+import random 
+import pdb
+
 def draw_letters():
-    pass
+
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    # create a list that will eventually be filled with 10 different letters
+    LENGTH_OF_HAND = 10
```

Comment:
It looks like this constant is only used in one place and only in this function. I see there's another constant later that's the same value. It might be a good idea to declare this outside of the function so that it can be used in all the functions.